docs/Usage.md

```diff
@@ -83,6 +75,11 @@ To activate the data transfer from the Modbus TCP Connector, proceed as followin
 
 - go to tab 'Assets & Connectivity' and add the variables, that were configured within the Modbus TCP Connector
 
-![DataServiceAdapter](/docs/graphics/DataService_Add.png)
+- ## Use Store Data in Common Configurator
+- (It´s necessary to have IIH Essentials installed)
+- open Define Data in Common Configurator
+- choose Modbus TCP Connector
+- Add asset, add tags to the asset and deploy
+

```

Comment:
Isn't it also necessary to select the "Storage" attribute here, if you want to store the data in IIH Essentials? Maybe an additional screenshot would be helpful.